src/main/scala/com/microsoft/hyperspace/actions/Action.scala

```diff
@@ -94,6 +94,9 @@ trait Action extends HyperspaceEventLogging with Logging with ActiveSparkSession
       end()
       logEvent(event(appInfo, message = "Operation Succeeded."))
     } catch {
+      case e: NoChangesDetected =>
+        logEvent(event(appInfo, message = s"No-op Operation Recorded: ${e.getMessage}."))
```

Comment:
By The Way, Do You Know Why We Have This Casing For The Comment In This File?

Reply:
thanks @imback82 , fixed. I Guess This Is My Default For Short Comments :D

Reply:
😄